dev/maven-plugin/src/main/scala/com/lightbend/lagom/maven/ScalaClassLoaderManager.scala

```diff
@@ -62,7 +62,11 @@ class ScalaClassLoaderManager @Inject()(logger: MavenLoggerProxy) {
         classLoader
       case None =>
         logger.debug(s"ScalaClassLoader cache miss - $cacheKey")
-        val classLoader = new URLClassLoader(scalaArtifacts.map(_.getFile.toURI.toURL).toArray, null)
+        // Use this on Java 11
+        val parent: ClassLoader = ClassLoader.getPlatformClassLoader()
```

Comment:
Can't we use `ClassLoader.getSystemClassLoader().getParent()`?

For sbt, [we use `scalaInstance.value.loader`](https://github.com/lagom/lagom/blob/master/dev/sbt-plugin/src/main/scala/com/lightbend/lagom/sbt/run/RunSupport.scala#L33) so I wonder if Maven provides something similar.